util_network.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import asyncio
+
+
+def parse_args():
+    """
+    Add command line optional arguments:
+     `-s`, `--server-hostname`: set your server's IP address.
+     `-m`, `--map`: map which will be played.
+    Return arguments.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-s", "--server-hostname", help="set custom server IP")
+    parser.add_argument("-m", "--map", help="set custom game map")
+    args = parser.parse_args()
+    return args
+
+
+def set_server_ip():
+    """
+    Return server IP: either custom, or 'localhost' if not filled in.
+    """
+    args = parse_args()
+    return args.connection if args.connection else "localhost"
```

Comment:
To je na mě složitý return :-D Nevím co je teda args.connection ale jinak si to mám představit jako podmínku, když se zadá adresa, tak je to ta adresa a jinak "localhost" že?